great-idea/css/index.css

```diff
@@ -62,3 +62,220 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+
+h1 {
+	font-size: 80px;
+}
+
+.container {
+	width: 880px;
+	margin: 0 auto;
+	border: 3px solid blue;
+  }
+
+header {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+	width: 100%;
+	border: 3px solid green;
+}
+
+header nav {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+	margin-top: 35px;
+	width: 75%;
+	border: 3px solid yellow;
+  }
+
+header nav a {
+	margin-right: 65px;
+	font-size: 14px;
+	color: gray;
+	text-decoration: none;
+	border: 3px solid red;
+  }
+
+header nav a:hover {
+	text-decoration: underline;
+}
+
+
+.cta {
+	margin-top: 100px;
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+	justify-content: space-evenly;
+	border: 3px solid red;
+	border-bottom: 1px solid black;
+}
+
+.cta-text {
+	text-align: center;
+	padding-top: 50px;
+}
+
+.cta-text button {
+	width: 150px;
+	font-size: 20px;
+	border: 1px solid black;
+}
+
+.cta img {
+	flex-direction: row;
+	justify-content: flex-start;
+	border: 3px solid purple;
+	margin-bottom: 60px;
+}
+
+.main-content .top-content {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-evenly;
+	margin-top: 30px;
+	
+}
+
+.main-content .bottom-content {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-around;
+}
+
+
+.middle-img {
+	margin-top: 30px;
+}
+
+.bottom-content {
+	border-bottom: 1px solid black;
+	padding-bottom: 30px;
+}
+
+.bottom-content .text-content {
+	width: 280px;
+	border: 2px solid teal;
+	margin-top: 30px;
+}
+
+.contact {
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+	border: 2px solid chartreuse;
```

Comment:
I like how you're using borders to troubleshoot and give yourself a visual indication of where things start and end. Just don't forget that the margin is outside the border -- that used to trip me up when I used the same approach.